### ISSUE TITLE
[css-anchor-position] Enable scroll position change for focused elements within anchored content

### ISSUE THREAD

**Issue by 5t3ph** [w3c/csswg-drafts#11110]
When content within an anchor positioned element gains focus in a way that should change the scroll position, no scroll position change occurs even when the anchored element is completely out of the viewport.

As seen in this CodePen, the anchored element has a link which should be scrolled to with the second tab stop when testing via keyboard tabbing. The scroll event happens appropriately for absolute, fixed, and sticky positioned non-anchored content as also demonstrated.

https://codepen.io/5t3ph/pen/abeGdeb

Expectation is that scroll position should adjust to reveal focused elements the same as regular document content. Not ensuring focused content is visible is a WCAG violation ([SC 2.4.11 - Focus Not Obscured](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html)) that the platform should default to resolving (acknowledging author styles/scripts can subsequently break again).

The solution should also account for bringing elements which gain focus back into view that may have been hidden via `position-visibility` to avoid obscuring keyboard focus that may remain in those elements.

**Comment by astearns** [comment]
@5t3ph this may just be a bug in Blink. Could you open an issue here for this?

https://issues.chromium.org/issues?q=is:open

**Comment by fantasai** [comment]
WPT tests for this: https://wpt.fyi/results/css/css-anchor-position?label=master&label=experimental&aligned&q=scroll-to-anchored-fixed

@5t3ph Is there anything to follow up with on the `position-visibility` point?
